plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceAuthModelDescriptor.java

```diff
@@ -42,6 +42,27 @@ public class DataSourceAuthModelDescriptor extends DataSourceBindingDescriptor i
 
     public static final String EXTENSION_ID = "org.jkiss.dbeaver.dataSourceAuth"; //$NON-NLS-1$
 
+    public enum DSPriority {
+        EXTRA_HIGH(1), // For necessary models
+        HIGH(2); // For priority models
```

Comment:
It looks strange. If high priority exists, there should also be normal priority somewhere

Reply:
And it is for the future. We can mark all our drivers with priority marks, but why do we need to do it without a special request?

Reply:
Ok, what do you suggest?

Reply:
Or
Normal = 1, Medium = 2,  High = 3, Urgent = 4